docs/source/getting-started/quickstart.rst

```diff
@@ -165,8 +165,26 @@ Stop/terminate a cluster
 =========================
 
 When you are done, run :code:`sky stop mycluster` to stop the cluster. To
-terminate a cluster instead, run :code:`sky down mycluster`.  Find more commands that
-manage the lifecycle of clusters :ref:`here <interactive-nodes>`.
+terminate a cluster instead, run :code:`sky down mycluster`.
+
+Find more commands that manage the lifecycle of clusters in the :ref:`CLI reference <cli>`.
+
+Scaling out
+=========================
+
+So far, we have used the `serverful` CLIs of SkyPilot, where the notion of clusters is central.
+This is useful for development since it is convenient to log into a cluster and perform debugging.
+
+When you are ready to scale out (e.g., run 10s or 100s of jobs), SkyPilot supports two options:
+
+- Queue jobs on a cluster (see :ref:`Job Queue <job-queue>`); or
```

Comment:
The reader just used `sky exec` in the quickstart above. Perhaps we should reference it here, hinting that `sky exec` is how you submit jobs to the job queue
```suggestion
- Queue jobs on a cluster with :code:`sky exec` (see :ref:`Job Queue <job-queue>`); or
```

Reply:
Good call, reworded.